package.json

```diff
@@ -8,7 +8,10 @@
 		"build": "tsc && cp src/index/transport/transport.avsc dist/index/transport",
 		"coverage": "nyc --report-dir=coverage --nycrc-path=test/nyc.opts mocha",
 		"coverage-systemtests": "nyc --report-dir=coverage --nycrc-path=systemtests/nyc.opts mocha --opts systemtests/mocha.opts",
-		"doc": "typedoc --ignoreCompilerErrors --out doc src/**/* --mode file",
+		"doc": "npm run doc:generate && npm run doc:sanitize:avsc && npm run doc:sanitize:node",
+		"doc:generate": "typedoc --gitRevision master --ignoreCompilerErrors --out doc src/**/* --mode file",
+		"doc:sanitize:avsc": "sed -E -i '' 's|'$PWD'/node_modules/avsc/types/index.d.ts:([0-9]*)|<a href=\"https://github.com/mtth/avsc/blob/master/types/index.d.ts#L\\1\">avsc/types/index.d.ts:\\1</a>|g' $(find doc -type f -regex '^doc/[a-zA-Z/.]*.html$')",
+		"doc:sanitize:node": "sed -E -i '' 's|'$PWD'/node_modules/@types/node/index.d.ts:([0-9]*)|<a href=\"https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/index.d.ts#L\\1\">node/index.d.ts:\\1</a>|g' $(find doc -type f -regex '^doc/[a-zA-Z/.]*.html$')",
```

Comment:
OPTIONAL: Seems to break the convention of hyphenated script-names. Maybe something like...
```
"doc": "typedoc bla bla...",
"postdoc": "npm run doc-fix-avsc && npm run doc-fix-node"
```